solutions/swb-reference/integration-tests/support/resources/projects/project.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AxiosResponse } from 'axios';
+import ClientSession from '../../clientSession';
+import Resource from '../base/resource';
+
+export default class Project extends Resource {
+  public constructor(id: string, clientSession: ClientSession, parentApi: string) {
+    super(clientSession, 'project', id, parentApi);
+  }
+
+  public async assignUserToProject(
+    userId: string,
+    requestBody: Record<string, string>
+  ): Promise<AxiosResponse> {
+    return this._axiosInstance.post(`${this._api}/users/${userId}`, requestBody);
+  }
+
+  protected async cleanup(): Promise<void> {
+    await this._setup.getDefaultAdminSession();
```

Comment:
should the project be deleted here?

Reply:
Yes, but delete API is not ready yet; this function should be revised and reviewed once CRUD projects is ready

Reply:
Please leave a TODO so it doesnt get overlooked